package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bigcommerce-cornerstone",
   "description": "The BigCommerce reference theme for the Stencil platform",
-  "version": "6.1.3",
+  "version": "6.1.4",
```

Comment:
@BC-Chkalov-Valerii you don't need to bump version here, it is done by Stencil during release process.

Reply:
@BC-krasnoshapka maybe it is acceptable if package related to ticket. In this case it looks more complete and we have more chances to find an issue during manual testing (imho)